homeassistant/components/foscam/camera.py

```diff
@@ -26,6 +27,7 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+    vol.Optional(CONF_RTSP_PORT, default=None): vol.Any(None, cv.port)
```

Comment:
Remove default `None` and use `dict.get` when accessing the config key instead.

Reply:
done :)